runtime/queries/apex/locals.scm

```diff
@@ -1,37 +1,48 @@
 ; declarations
 
 (class_declaration) @local.scope
+
 (method_declaration) @local.scope
+
 (constructor_declaration) @local.scope
+
 (enum_declaration) @local.scope
+
 (enhanced_for_statement) @local.scope
 
 ; if/else
 
 (if_statement) @local.scope
+
 (if_statement
   consequence: (_) @local.scope) ; if body in case there are no braces
+
 (if_statement
   alternative: (_) @local.scope) ; else body in case there are no braces
-
 ; try/catch
```

Comment:
Hmm, this empty line should ideally stay

Reply:
It should be better now.